code/game/objects/structures/cryopod.dm

```diff
@@ -0,0 +1,91 @@
+/////////////////CRYOPODS
+////////////////////////////////
+proc/justequip(var/mob/living/carbon/human/H, var/title, var/alt_title, var/outfit_type)
```

Comment:
Why there is no path for this proc?

Reply:
So that it can be used not only in cryopods, probably?